usecase/src/main/java/org/mobilitydata/gtfsvalidator/usecase/GetValidationUsecaseAll.java

```diff
@@ -0,0 +1,48 @@
+/*
+ *
+ *  * Copyright (c) 2020. MobilityData IO.
+ *  *
+ *  * Licensed under the Apache License, Version 2.0 (the "License");
+ *  * you may not use this file except in compliance with the License.
+ *  * You may obtain a copy of the License at
+ *  *
+ *  *     http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ *  * Unless required by applicable law or agreed to in writing, software
+ *  * distributed under the License is distributed on an "AS IS" BASIS,
+ *  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  * See the License for the specific language governing permissions and
+ *  * limitations under the License.
+ *
+ */
+
+package org.mobilitydata.gtfsvalidator.usecase;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Use case to retrieve the list of use cases to execute in order to process to validation of GTFS dataset
+ */
+public class GetValidationUsecaseAll {
```

Comment:
As discussed, it would be preferable to have something like `GetUsecaseCollectionForFile` taking the file name as a parameter to its execute method.